lib/ddtrace/contrib/http/circuit_breaker.rb

```diff
@@ -27,8 +27,8 @@ def datadog_http_request?(request)
           end
         end
 
-        def should_skip_distributed_tracing?(pin)
-          return !pin.config[:distributed_tracing] if pin.config && pin.config.key?(:distributed_tracing)
+        def should_skip_distributed_tracing?(client_config)
+          return !client_config[:distributed_tracing] if client_config && client_config.options.key?(:distributed_tracing)
 
           !Datadog::Tracing.configuration[:http][:distributed_tracing]
```

Comment:
Not for this PR, but I wonder if it would make sense to merge this back into instrumentation.rb, since it only gets used from there.

Reply:
Yeah, this particular integration code is a bit gnarly, and we should clean it up later.